TF_Service_dotNet/TouchFree/Interactions/PositionStabiliser.cs

```diff
@@ -132,8 +131,7 @@ void ShrinkDeadzone(Vector2 constrainedPositionDefault)
 
             if (previousConstraintVector != Vector2.Zero)
             {
-                float distanceAwayFromConstraint = Vector2.Dot(defaultPositionChange, previousConstraintVector) / previousConstraintVector.Length();
-                distanceAwayFromConstraint = Math.Max(0, distanceAwayFromConstraint) / 1000;
+                float distanceAwayFromConstraint = Math.Abs(Vector2.Dot(defaultPositionChange, previousConstraintVector) / previousConstraintVector.Length());
```

Comment:
I'd guess that if we're trying to shrink the Deadzone by a negative amount (I guess _growing_ the Deadzone) that we'd want to do nothing rather than continue to shrink it regardless?

Equally I don't think we want to allow growing the deadzone by these means as there is a mechanism to stop the shrinking once it hits a threshold that wouldn't be hit going larger.